### PR TITLE
update file

### PR DESCRIPTION
**Pull request by ferreiratiago**

### CODE REVIEW

File: src/file.go [hunk 1]
```diff
@@ -3,5 +3,5 @@ package main
 import "fmt"
 
 func main() {
-	fmt.Println("Hello, world!")
+	fmt.Println("Hello, world x1!")
```
Comment (padawancat) [inline src/file.go:6]: Why this?

**Review by padawancat — CHANGES_REQUESTED** [review]

File: src/file.go (hunk 1 above)
Comment (padawancat) [reply src/file.go:6]: Hum.

Comment (padawancat) [reply src/file.go:6]: Hello

**Comment by padawancat** [comment]
General comment.

**Comment by padawancat** [comment]
General comment 2.

**Review by padawancat — APPROVED** [review]

File: src/file.go [hunk 2]
```diff
@@ -3,5 +3,5 @@ package main
 import "fmt"
 
 func main() {
-	fmt.Println("Hello, world!")
+	fmt.Println("Hello, world x3!")
```
Comment (padawancat) [inline src/file.go:6]: Why this?

**Review by padawancat — CHANGES_REQUESTED** [review]

File: src/file.go (hunk 2 above)
Comment (ferreiratiago) [reply src/file.go]: /reviewpad resolve

**Comment by ferreiratiago** [comment]
**Reviewpad Report**

:scroll: **Executed actions**
```yaml
$addLabel("waiting-review")
```

**Comment by ferreiratiago** [comment]
**Reviewpad Report**

:scroll: **Executed actions**
```yaml
$rebase()
```

**Comment by ferreiratiago** [comment]
**Reviewpad Report**

**:bangbang: Errors**
* **Unconventional commit detected**: 'update workflow' (8b856e8d09c11c8f561af6fdf3c9f4e100ede0b9)
* **Unconventional commit detected**: 'update file' (4539451a2b7c294753ecc39a39f1426d310a7b06)
* **Unconventional commit detected**: 'update file' (2b5022f0c40ad6573c2e322d5ddf78f720a836f0)
* **Unconventional commit detected**: 'update file' (0b837d3457492d4d77ea75426a1900ba233466b8)
* **Unconventional commit detected**: 'add changes' (8b72055b676c5bcf8dbc5aaaf4611b46dfe48483)

**Comment by ferreiratiago** [comment]
**Reviewpad Report**

:scroll: **Executed actions**
```yaml
$rebase()
```

**Pull request closed without merging**